src/OpenTK/Platform/Android/AndroidDisplayDeviceDriver.cs

```diff
@@ -12,25 +12,37 @@ namespace OpenTK.Platform.Android
     internal class AndroidDisplayDeviceDriver : IDisplayDeviceDriver
```

Comment:
What did the android platform use before? It's been used successfully in the past, or at least that's what I assume based on what Xamarin has done.